### ISSUE TITLE
Group network outputs

### ISSUE THREAD

**Issue by lukeyeager** [NVIDIA/DIGITS#786]
Currently, DIGITS displays all network outputs in one big graph, and another separate one for the learning rate:

![outputs-all-together](https://cloud.githubusercontent.com/assets/687269/15550898/5d7737ae-2268-11e6-9d58-3d958f420c8f.jpg)

As you can see, that's pretty useless for more complicated networks with many outputs.

In another internal NVIDIA tool, we group network outputs by type (or by name?) and show them in separate graphs, with the learning rate super-imposed over each graph:

![outputs-separated](https://cloud.githubusercontent.com/assets/687269/15550924/7a820630-2268-11e6-843d-33692205d2f1.jpg)

I find that much nicer. I'd like to add this to DIGITS, but in such a way that it doesn't take up the whole page.

**Comment by TimZaman** [comment]
Maybe also fix the plot colors for validation, training and accuracy? Currently they are based on the the index of the data array it seems, and there's a discrepancy between Caffe and Torch in that:

## Caffe:
![image](https://cloud.githubusercontent.com/assets/7721540/15646748/c75fe18e-265f-11e6-8097-da4f98431099.png)

## Torch: 
![image](https://cloud.githubusercontent.com/assets/7721540/15646754/d1356940-265f-11e6-87d3-59daf64c3d96.png)

**Comment by lukeyeager** [comment]
> Currently they are based on the the index of the data array it seems, and there's a discrepancy between Caffe and Torch in that:

Yeah, that bothers me too.
